websites/B/bilibili/presence.ts

```diff
@@ -15,23 +7,26 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
   return [Math.floor(startTime / 1000), endTime];
 }
 
-var browsingStamp = Math.floor(Date.now() / 1000);
-
-var user: any;
-var title: any;
-var UID: any;
-var page: any;
-var currentTime: any,
+const presence = new Presence({ clientId: "639591760791732224" }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+  });
+const browsingStamp = Math.floor(Date.now() / 1000);
+let user: any, title: any, inread_title: any, UID: any, page: any;
+let video: any, videoDuration: any, videoCurrentTime: any, videoPaused: any;
```

Comment:
dont use any to declare, actually declare it like, string, number, HTMLElement, HTMLVideoElement, etc.